src/VCF_Tasks.cpp

```diff
@@ -43,11 +43,13 @@ bool run_read_adc2_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo&
 {
     adc_2.sample(); // Samples all eight channels.
     adc_2.convert(); // Converts all eight channels.
+    /*
```

Comment:
Remove this block comment. We'll tick the pedals system in a centralized task somewhere, so you don't have to do it.

Reply:
Don't resolve this comment unless you've resolved the comment. Remove the block comment.